### DISCUSSION TITLE
What is the difference between 0.11.X and 0.10.X

### DISCUSSION THREAD

**Discussion by sasan-ai** [milvus-io/milvus#4414] [Q&A and General discussion] [answered]
I would like to start using the Milvus in production. 
I'm planning to use it in a Kubernetes cluster but I'm not sure which one (i.e. v10 vs v11 )to choose from?

Can you please let me know the differences between the two and whether V11 is supported in a K8s cluster or not?

Thank you!

**Comment by shiyu22** [comment]
Hi, Milvus0.11.0 does not currently support clustering, so we recommend using 0.10.4.

You can refer to the [release notes](https://milvus.io/docs/v0.11.0/release_notes.md#RESTful) for version changes.

**Comment by yhmo** [comment, marked as answer]
0.11.x
- rewrite the storage module
- add data version control module: snapshot
- support multi-fields in a collection(including vector field and integer/float type field)
- support hybrid search by DSL search language

But 0.11.x not ready for clustering mode, yet.
